### PR TITLE
Linkify title in guide pages

### PR DESCRIPTION
**Pull request by sigvef**
This is now a link that goes to the frontpage:

image

### CODE REVIEW

**Comment by cristeahub** [comment]
Needs rebase

**Comment by sigvef** [comment]
ping

**Comment by cristeahub** [comment]
rebase problems again (sorry I merged the other one)

**Comment by sigvef** [comment]
ping

File: _sass/_myStyle.scss [hunk 1]
```diff
@@ -67,6 +67,11 @@ body.guide .container {
     margin: 1em auto;
     font-size: 17px;
 
+    h3 a {
```
Comment (cristeahub) [inline _sass/_myStyle.scss:70]: Did you mean `h3 { a {...`?

Comment (sigvef) [inline _sass/_myStyle.scss:70]: No, I meant `h3 a {`.

Comment (cristeahub) [inline _sass/_myStyle.scss:70]: Alright, just wanted to make sure

**Pull request merged**